airflow/config_templates/config.yml

```diff
@@ -1670,12 +1670,13 @@
         or insert it into a database (depending of the backend)
         This status is used by the scheduler to update the state of the task
         The use of a database is highly recommended
+        When not specified, sql_alchemy_conn with db+ prefix will be used for connection
         http://docs.celeryproject.org/en/latest/userguide/configuration.html#task-result-backend-settings
       version_added: ~
       type: string
       sensitive: true
       example: ~
-      default: "db+postgresql://postgres:airflow@postgres/airflow"
```

Comment:
i think it would be nice to move this to `example:` so it will be preserved but commented out

Reply:
Good point, changed